### DISCUSSION TITLE
Cannot configure collector

### DISCUSSION THREAD

**Discussion by SCzfdf** [netdata/netdata#18850] [Q&A]
# System
mac m1.

# Question
The collector cannot be identified correctly

# Process
1. Install netdata and start it
brew install netdata
2. Install redis and start it
brew install redis

# Try solutions
1. Restart netdata
brew services restart netdata

2. Configure the redis collector explicitly
2.1 visit http://localhost:19999/netdata.conf for the configuration file path
```
[directories]
# config = /opt/homebrew/etc/netdata
# stock config = / opt/homebrew/Cellar/netdata / 1.44.3 _10 / lib/netdata/conf. D
# log = /opt/homebrew/var/log/netdata
# web = / opt/homebrew/Cellar/netdata / 1.44.3 _10 / share/netdata/web
# cache = /opt/homebrew/var/cache/netdata
# lib = /opt/homebrew/var/lib/netdata
# lock = /opt/homebrew/var/lib/netdata/lock
# plugins = "/ opt/homebrew/Cellar/netdata / 1.44.3 _10 / libexec/netdata/plugins. The d" "/opt/homebrew/etc/netdata/custom-plugins.d"
# home = /Users/cgb
# registry = /opt/homebrew/var/lib/netdata/registry
# health config = /opt/homebrew/etc/netdata/health.d
# stock health config = / opt/homebrew/Cellar/netdata / 1.44.3 _10 / lib/netdata/conf. D/health. D
```

2.2 Configuring a redis collector
Add a configuration file in the following path
"/ opt/homebrew/Cellar/netdata / 1.44.3 _10 / libexec/netdata/plugins. D/redis. Conf." "
"/ opt/homebrew/Cellar/netdata / 1.44.3 _10 / libexec/netdata/plugins. D/go. D/redis. Conf." "
"/opt/homebrew/etc/netdata/redis.conf/go.d/redis.conf"
"/opt/homebrew/etc/netdata/redis.conf"

redis.conf
```conf
jobs:
- name: local
address: 'redis://@127.0.0.1:6379'
` ` `


2.3 Restarting netdata
brew services restart netdata

**Comment by SCzfdf** [comment]
Found the reason why plugs directory doesn't have go.plugs. But I installed them with brew

**Comment by Yakhyobek1997** [comment]
check if Redis is running on your system
 
 brew services list
 
 You should see something like = redis         started

You can also check if Redis is listening on the correct port (6379): redis-cli ping
